src/test/java/PawnTest.java

```diff
@@ -0,0 +1,22 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PawnTest {
+
+    @Test
+    @DisplayName("지정된 색상의 폰이 생성되어야 한다.")
+    void create() {
+        final String white = "white";
```

Comment:
```suggestion
        final String  WHITE = "white";
```
취향이겠지만 저는 대문자를 선호합니다.